index.js

```diff
@@ -129,25 +111,23 @@ const HOME_MENU_BUTTONS_ARRAY = [
     {
         id: ELEMENT_IDS.digitalProgrammeListTab,
         innerText: "Digital Programme List",
-        onclick: renderDigitalProgrammeList
+        onclick: renderNewScreen
```

Comment:
As we said, this onClick should be moved to .forEach

Reply:
Solved